DataCapturing.podspec

```diff
@@ -41,6 +41,8 @@ This framework can be included by your App if you are going to capture sensor da
   s.source_files = 'DataCapturing/Source/**/*{.h,.m,.swift}'
   s.resources = 'DataCapturing/Source/**/*{.xcdatamodeld,.xcdatamodel,.xcmappingmodel}'
 
+  s.script_phase = { :name => 'Set version and build', :script => "#!/bin/bash\n\n# Determine plist path\nplist=${BUILT_PRODUCTS_DIR}/${INFOPLIST_PATH}\n# Determine current version\nversion=$(/usr/libexec/PlistBuddy -c 'Print CFBundleShortVersionString' \"$plist\")\n# Get date of last commit\ncommitDate=$(date -r `git log -1 --date=short --pretty=format:%ct` +%Y%m%d)\n# Get current date/time\nbuildDate=$(date +%Y%m%d%H%M)\n\n# Set display version to existing version + commitDate\n/usr/libexec/PlistBuddy -c \"Set :CFBundleShortVersionString $version-$commitDate\" \"$plist\"  # Version number\n\n# Set build number to current date/time\n/usr/libexec/PlistBuddy -c \"Set :CFBundleVersion $buildDate\" \"$plist\"  # Build number\n\necho \"Version: $version-$commitDate\"\necho \"Build:   $buildDate\"\n" }
+
```

Comment:
This is the actual script included from RM. All the other changes below ensure that CocoaPods (which is the substitude for Gradle) is capable of actually running and building he project outside an XCode IDE.